IPython/utils/importstring.py

```diff
@@ -23,17 +24,17 @@ def import_item(name):
     mod : module object
        The module that was imported.
     """
-    
+
     parts = name.rsplit('.', 1)
     if len(parts) == 2:
         # called with 'foo.bar....'
         package, obj = parts
-        module = __import__(package, fromlist=[obj])
+        module = import_module(package, fromlist=[obj])
```

Comment:
I think this is the only one that isn't working. `fromlist` isn't supported in `import_module`, so we need to add some logic for importing objects. I'm not quite sure what the most obvious new-syntax for "import a.b.c, which might be a module or an object", since it will depend on what it is.

My guess is the best we can do is approximately:

```python
package, obj = parts
try:
    return import_module(name) # full name is a module
except ImportError:
    # not a module, maybe an attr on the parent module
    module = import_module(package)
    return getattr(module, obj)
```